crates/swift-bridge-macro/src/lib.rs

```diff
@@ -30,5 +30,6 @@ mod ui_tests {
     fn ui() {
         let t = trybuild::TestCases::new();
         t.compile_fail("tests/ui/*.rs");
+        t.pass("tests/ui-pass/*.rs");
```

Comment:
`crates/swift-integration-tests` can be used to make sure that things compile

Then (when applicable) we call the things from the `SwiftIntegrationTestRunner`

https://github.com/chinedufn/swift-bridge/tree/d05c6de93ac1c875adf25e2ed793d9dd86cb1f52/SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests